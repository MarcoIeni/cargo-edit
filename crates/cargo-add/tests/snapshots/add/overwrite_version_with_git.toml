bin.name = "cargo-add"
args = ["add", "versioned-package", "--git", "git://git.git", "-Zgit"]
status = "success"
stdout = ""
stderr = """
      Adding versioned-package (git) to optional dependencies.
"""
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
