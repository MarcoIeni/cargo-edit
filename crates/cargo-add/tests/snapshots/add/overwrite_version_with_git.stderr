      Adding versioned-package (git) to optional dependencies.
